tests/test_errors.py

```diff
@@ -193,6 +193,7 @@ def test_json_body_parameter_validation(self):
             path="/stuffs", data={"foo": "one", "bar": "not-an-email"}
         )
         expected = messages.body_validation_failed._asdict()
+        expected.pop("rebar_error_code")
```

Comment:
Why do we need to pop that key now?

Reply:
I fixed this now, it was actually related to this typo https://github.com/plangrid/flask-rebar/pull/300#discussion_r1435785217 - I think we actually don't need that line of code at all, so that way it passes an ErrorMessage object (rather than the `message` which is just a string) and we get the full detail in the response including the `rebar_error_code`.